tooling/v8-snapshot/test/fixtures/iterative/templates/intermediate-norewrite.js

```diff
@@ -0,0 +1,250 @@
+require ('./norewrite')
+
+// Generated by CoffeeScript 1.9.3
+(function() {
```

Comment:
Is this all test code? What part of it makes it 'norewrite', should we reduce this to just that?

Reply:
Updated: [`47fd989` (#23692)](https://github.com/cypress-io/cypress/pull/23692/commits/47fd9891fd645ccbe7085433d66f1997d2ee75f2)